packages/cascara/src/atoms/Title/Title.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { Role } from 'reakit';
 
-import { Boundaries } from '../../system-components';
-import { Flex } from '../../atoms';
+import Boundaries from '../../atoms/Boundaries';
+import Flex from '../../atoms/Flex';
```

Comment:
@lordpixel are we importing directly versus a sub export to resolve circular dependencies? 

I am asking because in packages/cascara/src/modules/ModuleErrorBoundary/ModuleErrorBoundary.js below we are importing a sub member. If we need to import directly instead of the sub member import, let's be consistent about which style we use.

IMO if it is going to make us less prone to circular dependency errors. This might also mean we should rethink having an index.js in each section directory where we export all defaults so it becomes impossible to import from the directory.

Let me know if there is a reason for the difference in style here before we make some larger determinations.

Reply:
Yes @brianespinosa, we are resolving circular dependencies.